examples/basic/package.json

```diff
@@ -14,26 +14,24 @@
     "@react-native-picker/picker": "^1.9.11",
     "react": "18.2.0",
     "react-native": "0.72.5",
-    "react-native-video": "../..",
     "react-native-windows": "0.63.41"
   },
   "devDependencies": {
-    "@babel/core": "^7.20.0",
-    "@babel/preset-env": "^7.20.0",
-    "@babel/runtime": "^7.20.0",
+    "@babel/core": "^7.22.10",
+    "@babel/preset-env": "^7.22.10",
+    "@babel/runtime": "^7.22.10",
     "@react-native/eslint-config": "^0.72.2",
     "@react-native/metro-config": "^0.72.11",
     "@tsconfig/react-native": "^3.0.0",
     "@types/react": "^18.0.24",
-    "@types/react-test-renderer": "^18.0.0",
     "@types/react-native-video": "^5.0.15",
```

Comment:
We should remove "@types/react-native-video" as typescript is now supported ?!

```suggestion
```

Reply:
Ops, it hid from me 😄